### PR TITLE
GUI-1688 Disallow use of 12-digit Account Numbers as Account Aliases

### PR DESCRIPTION
**Pull request by coyote240**
https://eucalyptus.atlassian.net/browse/GUI-1688

More unit tests running than there were, but I'm hung up on testing the form validation.  Broken tests disabled for now.

### CODE REVIEW

**Comment by dkavanagh** [comment]
hal, retest this please

**Comment by coyote240** [comment]
hal, retest this please

**Comment by aliceh** [comment]
@coyote240 I looked at the  test logs: for some reason the test user can not log in. Namely, after account, user name and password are typed in and login button is clicked the dashboard does not load as expected.

**Comment by aliceh** [comment]
@dkavanagh @coyote240 looks like the passwords for standard ui-test-acct users have expired on the CI cloud. I will extend expiration date now.

**Comment by aliceh** [comment]
guitester, retest this please

**Comment by aliceh** [comment]
retest this, please

File: eucaconsole/static/js/jasmine-spec/spec_account_new.js [hunk 1]
```diff
@@ -56,4 +56,22 @@ describe("AccountPage", function() {
             expect(scope.setWatch).toHaveBeenCalled();
         });
     });
+
+    xdescribe("Form Validation", function () {
```
Comment (kamalgill) [inline eucaconsole/static/js/jasmine-spec/spec_account_new.js]: @coyote240: Should this test suite be enabled or removed?

**Pull request merged**